package.json

```diff
@@ -8,8 +8,9 @@
     "preinstall": "npx only-allow pnpm",
     "dev": "webpack serve",
     "build": "webpack --mode production",
-    "stats": "webpack --mode production --profile --json > stats.json",
-    "bundle-analysis": "pnpm stats && pnpm dlx webpack-bundle-analyzer stats.json",
+    "bundle-stats": "webpack --mode production --profile --json > stats.json",
+    "show-bundle-analysis": "pnpm dlx webpack-bundle-analyzer stats.json",
```

Comment:
Why should that help?
You almost always need to create a new stat file when showing the analysis. So it doesn't make sense to have a script running on an existing stats file. The `stats` is also short enough and doesn't need to be changed to `bundle-stats` IMO.

Reply:
Done